src/runtime/auto-prompt-manager.ts

```diff
@@ -961,6 +971,37 @@ export class AutoPromptManager {
     return true;
   }
 
+  /**
+   * Checks Intervention eligibility, used to filter interventions in a given
+   * funnel.
+   */
+  private checkOrchestrationEligibility_(
+    orchestration: InterventionOrchestration,
+    eligibleActionIds: Set<string | undefined>,
+    article: Article
+  ): boolean {
+    if (!eligibleActionIds.has(orchestration.configId)) {
+      return false;
+    }
+    if (
+      this.isRepeatableIntervention_(orchestration.type) &&
```

Comment:
Do we need to keep maintaining this list in the client side or can we defer it to the server-side such that repeatability values are set up properly for repeatable actions types?

Reply:
Agreed, removing this check and deferring to the server to handle repeatabiltiy type checks.